pkg/agent/openflow/pipeline.go

```diff
@@ -1652,6 +1654,26 @@ func (c *client) conjunctionActionDenyFlow(conjunctionID uint32, table binding.T
 		Done()
 }
 
+func (c *client) conjunctionActionPassFlow(conjunctionID uint32, table binding.Table, priority *uint16, disposition uint32, enableLogging bool) binding.Flow {
+	ofPriority := *priority
+	nextTable := IngressRuleTable
+	tableID := table.GetID()
+	if _, ok := egressTables[tableID]; ok {
+		nextTable = EgressRuleTable
+	}
+	flowBuilder := table.BuildFlow(ofPriority).MatchConjID(conjunctionID)
+	if enableLogging {
+		customReason := CustomReasonLogging
+		flowBuilder = flowBuilder.
+			Action().LoadRegMark(DispositionPassRegMark).
+			Action().LoadToRegField(CustomReasonField, uint32(customReason)).
+			Action().SendToController(uint8(PacketInReasonNP))
```

Comment:
I don't see conjunctionID is loaded anywhere. How is it going to be logged without the rule ID?

Reply:
From my understanding, the logger does not need conjID loaded? Adding @qiyueyao to confirm

Reply:
Turns out you are absolutely correct. Added the conjunction to TF reg. Verified that it works in logging:
```
2021/11/23 00:15:54.327084 AntreaPolicyIngressRule AntreaClusterNetworkPolicy:cnp-test-pass Pass 14900 10.10.1.5 10.10.1.6 60 TCP
2021/11/23 00:16:05.335738 AntreaPolicyIngressRule AntreaClusterNetworkPolicy:cnp-test-pass Drop 14899 10.10.1.5 10.10.1.3 60 TCP
```